page.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <script src="https://kit.fontawesome.com/138cf4be1d.js" crossorigin="anonymous"></script>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="./style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+  <title>Akpuru Solomon Website</title>
+</head>
+
+<body>
+  <div id="container">
+   <header>
+      <div class="topnav" id="myTopnav">
+        <a class="logo" href="#">Akpuru Solomon</a>
+        <a href="#works" class="active">Portfolio</a>
+        <a href="#about">About</a>
+        <a href="#contact">Contact</a>
+        <a class="bars" href="#"><i class="fa fa-bars"></i></a>
+      </div>
+    </header>
+    <section id="opening">
+      <div id="intro_word">
+      <h1 id="title">Hey There. I’m Solomon</h1>
+      <h2 id="job">I’m a Software Developer</h2>
+      <p id="description">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+        don’t hestiate to contact me.</p>
+        </div>
+      <ul id="social">
+        <li><a href="https://github.com/Solobarine"><i class="social-icons fa-brands "></i><img src="/Image/social media/github.png" alt="my github handle" width="20" height="20"></a></li>
+        <li><a href="https://www.linkedin.com/in/solomnakpuru/"><i class="social-icons fa-brands "></i><img src="/Image/social media/linkedin.png" alt="my linkedin handle" width="20" height="20" ></a></li>
+        <li><a href="#"><i class="social-icons fa-brands "></i><img src="/Image/social media/angelist.png" alt="my angelist handle" width="20" height="20"></a></li>
+        <li><a href="https://twitter.com/SolomonAkpuru"><i class="social-icons fa-brands "></i><img src="/Image/social media/twitter.png" alt="my twitter handle" width="20" height="20"></a></li>
+        <li><a href=""><i class="social-icons fa-brands fa-microverse"></i><img src="/Image/social media/Vector_m.png" alt="my m handle" width="20" height="20"></a></li>
+      </ul>
+    </section>
+
+    <section id="work">
+      <h2 id="work_title">My Recent Works</h2>
+
+      <div id="projects">
+
+        <div id="frames">
+          <div id="recent-img"><img src="./image/Img placeholder.png" alt=""></div>
```

Comment:
- [ ] Please avoid naming your assets files with spaces. It would be safe if you name your multi word files with `-` between the words. The `Img placeholder.png` is not loading because it has got case conflict. Please make sure your website loads all the assets written in the code. 

![Screenshot from 2022-07-09 12-25-31](https://user-images.githubusercontent.com/12524453/178104418-86c877d5-5d9e-405f-ba77-4cfdf72ca567.png)